post_installation_configuration/coreos-layering.adoc

```diff
@@ -22,13 +27,28 @@ Installing realtime kernel and extensions RPMs as custom layered content is not
 
 As soon as you apply the custom layered image to your cluster, you effectively _take ownership_ of your custom layered images and those nodes. While Red Hat remains responsible for maintaining and updating the base {op-system} image on standard nodes, you are responsible for maintaining and updating images on nodes that use a custom layered image. You assume the responsibility for the package you applied with the custom layered image and any issues that might arise with the package.
 
-To apply a custom layered image, you create a Containerfile that references an {product-title} image and the RPM that you want to apply. You then push the resulting custom layered image to an image registry. In a non-production {product-title} cluster, create a `MachineConfig` object for the targeted node pool that points to the new image.
+There are two methods for deploying a custom layered image onto your nodes:
+
+On-cluster layering:: With xref:../post_installation_configuration/coreos-layering.adoc#coreos-layering-configuring-on_coreos-layering[on-cluster layering], you create a `MachineOSConfig` object where you include the Containerfile and other parameters. The build is performed on your cluster and the resulting custom layered image is automatically pushed to your repository and applied to the machine config pool that you specified in the `MachineOSConfig` object. The entire process is performed completely within your cluster. 
++
+--
+:FeatureName: On-cluster image layering
+include::snippets/technology-preview.adoc[]
+--
+
+//Naming comes from enhancement doc: https://docs.google.com/document/d/1QSsq0mCgOSUoKZ2TpCWjzrQpKfMUL9thUFBMaPxYSLY/edit#heading=h.bn17fuch1b5i
```

Comment:
Maybe leave this comment out of the source and just have this internal gdoc mentioned in the linked Jira issue.